api_tests/dry/sanity.ts

```diff
@@ -88,20 +89,42 @@ setTimeout(async function() {
             expect(res.body.peers).to.have.length(0);
         });
 
-        it("[Alice] Returns its peer ID when you GET /", async () => {
+        it("[Alice] Response for GET / is a valid siren document", async () => {
```

Comment:
The PR title and the commit does not mention changes on `/`.
Please update README and fix description.